src/foam/nanos/fs/File.js

```diff
@@ -64,9 +64,30 @@ foam.CLASS({
       name: 'dataString',
       documentation: 'File converted to base64 string'
     },
+    {
+      class: 'String',
+      name: 'image',
+      createVisibility: 'HIDDEN',
+      updateVisibility: 'RO',
+      readVisibility: 'RO',
+      transient: true,
+      storageTransient: true,
```

Comment:
transient is both networkTransient and storageTransient, so only one of these are needed.